decidim-core/lib/decidim/form_builder.rb

```diff
@@ -290,8 +290,11 @@ def resources_select(name, collection, options = {})
     #
     # Returns a String.
     def scopes_picker(attribute, options = {})
-      id = "#{@object_name}_#{attribute}"
-      id = "#{self.options[:namespace]}_#{id}" if self.options.has_key?(:namespace)
+      id = if self.options.has_key?(:namespace)
```

Comment:
can you document this new :namespace opcion please?

Reply:
The `:namespace` option was there before I refactored those lines, in any case I'll add it to the docs. 😀